dev/AppLifecycle/Association.cpp

```diff
@@ -54,13 +54,19 @@ namespace winrt::Microsoft::ProjectReunion::implementation
         return path;
     }
 
-    std::wstring ComputeAppId()
+    std::wstring ComputeAppId(const std::wstring& customSeed)
     {
         // PRF = Project Reunion Framework.  A simple prefix to help identify ProgIds we compute here.
```

Comment:
SUGGESTION: Change PRFblah to Microsoft.AppLifecycle.blah

A few reasons:
* PRF is a bit cryptic and ambiguous
* Last year the point was made several times not to put "ProjectReunion" into code; I don't recall data constracts being mentioned though they'd seem to meet the spirit of the guidance.
* ProgIDs have large length limits (255?)

Reply:
I'm not opposed to finding a different prefix that makes this more readable and/or more compliant with any thinking around using ProjectReunion in code.  The binary is named this currently, and I know that is staying while we will be changing the namespace.  If this specific instance is not required to change due to whatever ruleset governs that, then we only have readability to go off of for reasons to change this.  

That being said, using the namespace isn't a good option either.

Also, ProgIds do not have a large limit.  They are max of 39.  https://docs.microsoft.com/en-us/windows/win32/com/-progid--key